roles/builder/defaults/main.yml

```diff
@@ -15,4 +15,6 @@ builder_compose_customizations:
     description: "test user"
     password: "openshift"
     key: "{{ builder_pub_key }}"
-    groups: '["users", "wheel"]'
+    groups: 
```

Comment:
Lint is complaining about a trailing space on line 18

Reply:
fixed!